.github/actions/apk/action.yml

```diff
@@ -0,0 +1,61 @@
+name: apk
+runs:
+  using: composite
+  steps:
+    - shell: sh
```

Comment:
Any particular reason to use sh? We use bash everywhere else.

Reply:
Yes, it's because there is no `bash` in Alpine by default

Reply:
Ah, gotcha.